static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -268,4 +268,9 @@
         }
     }
 
+    .media-primary & {
+        @include mq($from: mobile, $until: mobileLandscape) {
+            margin-top: -1rem;
```

Comment:
@NataliaLKB cool, OK I'll take a look at the cleaner and see about adding --main-media to the atom. I'm happy to refactor this file to remove class concatenation! Regarding nesting how far should we go? Are we now suggesting we remove all nesting? Or is it OK to keep nesting to denote parent/child relationships?